packages/flutter_tools/test/integration.shard/cache_test.dart

```diff
@@ -0,0 +1,144 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
+
+import 'dart:io' as io show ProcessSignal;
+
+import 'package:file/file.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/base/terminal.dart';
+import 'package:flutter_tools/src/cache.dart';
+import 'package:process/process.dart';
+
+import '../src/common.dart';
+import '../src/context.dart';
+import 'test_utils.dart';
+
+class FakeLogger extends Logger {
```

Comment:
Please use the BufferLogger class instead

Reply:
Ahh, that's what it's called.  I was looking for something like that (expected it to be called "Fake" something). OK.

Reply:
OK, Fixed.  PTAL